packages/trader/src/App/Components/Layout/Header/account-actions.jsx

```diff
@@ -67,30 +69,31 @@ export class AccountActions extends Component {
                             window.open(urlFor('user/accounts', undefined, undefined, true));
                         }}
                     />}
-                    {/* {!is_virtual && */}
-                    {/* <ToggleCashier */}
-                    {/*    active_tab={active_cashier_tab} */}
-                    {/*    className='acc-info__button' */}
-                    {/*    toggleCashier={toggleCashierModal} */}
-                    {/*    is_cashier_visible={is_cashier_modal_on} */}
-                    {/* /> */}
-                    {/* } */}
-                    {!(
-                        is_virtual
-                    ) && // TODO: remove this when cashier pop up is ready
-                    <Button
-                        id='dt_deposit_button'
-                        className='btn--primary btn--primary--orange acc-info__button'
-                        has_effect
-                        text={localize('Deposit')}
-                        onClick={() => {
-                            window.open(urlFor('cashier', undefined, undefined, true),
-                                '_blank',
-                                'noopener',
-                                'noreferrer',
-                            );
-                        }}
-                    />}
+                    {!is_virtual &&
+                    <ToggleCashier
+                        active_tab={active_cashier_tab}
+                        className='acc-info__button'
+                        toggleCashier={toggleCashierModal}
+                        is_cashier_visible={is_cashier_modal_on}
+                        is_payment_agent_visible={is_payment_agent_visible}
+                    />
+                    }
+                    {/* {!( */}
+                    {/*    is_virtual */}
+                    {/* ) && // TODO: remove this when cashier pop up is ready */}
```

Comment:
Are we not ready to just remove this yet?

Reply:
no, whitelisting and email templates are pending